docker/oap/Dockerfile.oap

```diff
@@ -16,7 +16,23 @@
 
 ARG BASE_IMAGE='adoptopenjdk/openjdk8:alpine'
 
-FROM apache/skywalking-base:8.1.0-es6 AS cli
+FROM golang:1.14 AS cli
+
+ARG COMMIT_HASH=9f267876493943716434fdaa30047a14c0b5b2d9
```

Comment:
How do we deal with cli in docker release? We can't build docker file based on dev version.

Reply:
We need an Apache release CLI(0.7.0) for the 8.7 docker release. @kezhenxu94

Reply:
Hi @fgksgf, do you have time to release CLI 0.7.0?

Reply:
> Hi @fgksgf, do you have time to release CLI 0.7.0?

Ok, I will release it tonight